draft-issuance.md

```diff
@@ -0,0 +1,175 @@
+```
+ZIP: 
+Title: Smooth Out The Block Subsidy Issuance
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+Status: Draft
+Category: Consensus
+Created: 2023-08-23
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”,
+and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+"Network upgrade" - to be interpreted as described in ZIP 200. [2]
+
+“Block Subsidy” - to be interpreted as described in ZIP TBD [3]
+
+“Issuance” - to be interpreted as described in ZIP TBD [3]
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZsfBalanceAfter(h)`” is the total ZEC available in the Zcash Sustainability Fund (ZSF) after the transactions
+in block `h`, described in ZIP #TODO reference#. The Sustainability Fund is used to pay out Block Subsidies
+from unmined ZEC, and other fund deposits.
+
+“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
+
+"`NUMBER_OF_BLOCKS_IN_4_YEARS`" = `(365.25 * 24 * 60 * 60/75 * 4)`, or `1_683_072`
+
+# Abstract
+
+This ZIP proposes a change to how nodes calculate the block subsidy.
+
+Instead of following a step function around the four-year halving cycle inherited
+from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
+scheme would approximate the current 4 year cycle, and results in the last
+zatoshi being issued in around 114 years.
```

Comment:
Nitpick: there is no cycle after this ZIP, it's a smooth curve. And they are currently called "halving intervals".

```suggestion
Instead of following a step function around the 4-year halving intervals inherited
from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
scheme would approximate the current issuance over 4-year intervals, and results in the last
zatoshi being issued in around 114 years.
```